dtcm/dominion/map.xml

```diff
@@ -120,10 +98,10 @@
         <cuboid min="-95.5,0,-5.5" max="-45.5,1,-39.5"/> <!-- Crescent Left A -->
         <cuboid min="-58,0,-39.5" max="-93,1,-50"/> <!--Crescent Left B -->
     </union>
-    <apply enter="only-blue" region="blue-spawn" message="You may not enter the opponent's spawn!"/>
-    <apply enter="only-red" region="red-spawn" message="You may not enter the opponent's spawn!"/>
-    <apply block-break="only-iron" block-place="deny-players" region="spawns" message="Only iron blocks may be broken in the spawns!"/>
-    <apply block="deny(void)" region="void-area" message="You may not build in the void area!"/>
+    <apply enter="blue" region="blue-spawn" message="You may not enter the opponent's spawn!"/>
+    <apply enter="red" region="red-spawn" message="You may not enter the opponent's spawn!"/>
```

Comment:
For clarity I think we should rename the teams to `blue-team` and `red-team` in this instance